.circleci/config.yml

```diff
@@ -24,6 +24,16 @@ references:
           name: yarn install and dependencies
           command: |
             yarn install
+      - run: 
+          name: Check Dependencies
+          command: |
+            sudo apt install python3-pip
+            pip3 install --user virtualenv
+            virtualenv venv
+            python3 --version
+            source venv/bin/activate
+            pip3 install pathlib
+            python3 scripts/identify_dependencies.py
```

Comment:
do you have to do this? instead just install in the global context ie

sudo apt install python3-pip
pip3 install pathlib
 python3 scripts/identify_dependencies.py